content/work/adobe.md

```diff
@@ -39,4 +96,139 @@ summary: |
+</figure>
+
+
+### Audio & Nuxt.js
+
+Audio implementation provided an interesting challenging. First, we tried Web Audio API and a Nuxt.js plugin that automatically played and paused the sounds on each page of the game. When we tested it, the sounds played simultaneously. Uh oh! We switched to HTML5 Audio and added an ID for each question tag. The correct sound played when the ID and tag matched. Yay!
+
+During testing across different browsers and devices, we discovered that our audio solution didn’t work correctly everywhere — returning auto-play errors. We had to get clever. Using [HTML5 audio] made recovery trivial if autoplay failed.
+
+```js
+controlSound(on) {
+  const bg = this.$refs.audioBackground;
+  if (bg) {
+    on ? bg.play().catch(() =>  
+    this.toggleSetting(this.sound)) : bg.pause();
+  }
+},
+```
+
+It was fun collaborating with audio specialist, Rob Taliesin Owen of [Potion SoundWorks]. Adding sound elevated the whole experience of game play. 
+
+[html5 audio]: https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide
+[Potion SoundWorks]: https://potionsoundworks.com/
+
+### CSS Custom Properties
+
+Each question in the game has nine options and each option consists of the same identical image plus a unique font. Typefaces have a wide array of letter heights and widths. When we placed the typefaces on top of the images, some barely filled the space, some hung off the edges of the illustrations. CSS Custom Properties to the rescue! Custom Properties made it possible to adjust individual typefaces.
+
+{{ embed.figure(
+  data=before,
+  caption='By default, a single shared font-size does not work well across different fonts.'
+) }}
+
+First, we added the `svg-adjust` custom preoerty with js or something? ...
+
+```js
+computed: {
+  fontStyles() {
+    return {
+      '--svg-adjust': this.font.adjust,
+    };
+  }
+}
+```
+
+We set a global font-size using a CSS calc statement. The first value was the `--svg-base` custom property for the set of illustrations (with a fallback of 4em). We multiplied that by the `--svg-adjust` custom property of each individual font-family used in a set of illustrations (or 1 if it wasn't assigned).
+
+```css
+/* Global CSS */
+.svg-text {
+  font-size: calc(var(--svg-base, 4em) * var(--svg-adjust, 1));
+}
+```
+
+In the calligraphic page's Vue file, we set `--svg-base` scoped to this page only:
+
+```css
+<style lang="scss" scoped>
+.svg-text {
+  --svg-base: 87px;
+}
+</style>
+```
+
+Each page has a markdown file that lists the font families 
+along with some additional data. This is where we are 
+able to fine-tune each font-size per font-family.
+
+```md
+  - name: Chapman Bold Extended Italic
+    adjust: .675
+```
+
+{{ embed.figure(
+  data=compare,
+  caption='Before and after',
+  class='extend-small size-quarter'
+) }}
+
```

Comment:
in order to avoid an empty paragraph before and after this figure, what would I need to do?

Reply:
This is still an issue 🤔

Reply:
I'm not sure, but I would guess this isn't the only place it's happening. Maybe worth opening a new card to look into? Unless it's a blocker for this case study?

Reply:
@mirisuzanne made a new card: https://trello.com/c/PkgEjxR2/122-around-every-embedfigure-type-of-macro-it-adds-an-empty-p-p-before-and-after 

I see it in other places as well